api/configuration/configuration.go

```diff
@@ -54,6 +54,7 @@ type Configuration struct {
 	ValuesPath             string          `json:"values_path"`
 	ViewsPath              string          `json:"views_path"`
 	PhonebookPath          string          `json:"phonebook_path"`
+	RrdRootPath            string          `json:"rrd_path"`
```

Comment:
we can rename it to `RrdPath` ?

Reply:
I chose `RrdRootPath` to not confuse it with `rrdFilePath` variable [used in rrd.go](https://github.com/nethesis/nethvoice-report/pull/14/files/37ce396bd824807de39a400e9ed77e828a7e103c#diff-d57c2910b18d24bc4274f59e29c20f7e9bfcd056d44d5916990c716704a62baeR30).
Do you think `RrdPath` is better anyway?